bin/replication.js

```diff
@@ -47,7 +48,7 @@ function _createSetupReplication(command, options, log) {
         target: {
             bucket: targetBucket,
             credentials: targetCredentials,
-            hosts: new RoundRobin(destination.bootstrapList[0].servers),
+            hosts: new RoundRobin(destinationEndpoint.servers),
```

Comment:
if destination is `aws_s3`, this could be a problem since there is no `servers` attribute on it

This may require changes here and also in class `SetupReplication` since it relies on `RoundRobin` objects for the `hosts` attribute

Reply:
You're right, pushed an update.